README.md

```diff
@@ -89,6 +89,21 @@ Once you have either downloaded the VSIX file or created it you can now install
 
 Please refer to the Wiki page at https://github.com/apache/daffodil-vscode/wiki
 
+To debug a daffodil schema, choose open a folder from the File menus, then select a dfdl.xsd file. Once the file loads in VS Code, then set a desired breakpoint.
+![Setting Breakpoints](./images/SetBreakpoint.gif)
+
+To start debugging, click on the debug button on the side panel, then click the RUN AND DEBUG button with the dropdown set to Ask for file name.
+![Debugging](./images/StartDebug.gif)
```

Comment:
This GIF goes really fast, its hard to tell what it's actually trying to show. Does this add anything new that's not on the wiki?

Reply:
I can redo the GIFs, but as in the previous comment I'm not sure that is what you are asking for.